packages/commands/src/index.ts

```diff
@@ -553,7 +566,16 @@ export class CommandRegistry {
     // If there is an exact match but no partial match, the exact match
     // can be dispatched immediately. The pending state is cleared so
     // the next key press starts from the default state.
-    if (exact && !partial) {
+    if (
+      exact &&
+      !partial &&
+      !(
+        exact.keys.toString() === 'Alt' ||
+        exact.keys.toString() === 'Ctrl' ||
+        exact.keys.toString() === 'Shift' ||
+        exact.keys.toString() === 'Alt Shift'
+      )
+    ) {
```

Comment:
```suggestion
    const layout = getKeyboardLayout();
    const exactKeys = exact?.keys.toString().split(' ');
    if (
      exact &&
      !partial &&
      !exactKeys.every(key => layout.isModifierKey(key))
    ) {
```
It could be more generic and check all modification key combinations (there is probably a better implementation than this).